script.js

```diff
@@ -23,34 +27,100 @@ allShows.sort((a, b) => {
   }
   return 0;
 });
-// console.log(allShows);
 
-// ------------------Fetch data from API----------------------------
+// ------------------Fetch data from API--------------------------
 
-function getAllEpisodes(url) {
-  fetch(url)
-    .then((response) => response.json())
-    .then((data) => {
-      allEpisodes = Array.from(data);
-      makePageForEpisodes(data);
-      makeEpisodeMenu(data); //(allEpisodes);
-    })
-    .catch((error) => console.log(error));
+function loadShows(url) {
+  return fetch(url).then((response) => response.json());
 }
 
+//-------------------Setup the page onload------------------------
+
 function setup() {
-  makePageForEpisodes(allEpisodes);
+  hideEpisodeParts();
+  makePageForShows(allShows);
   makeTvShowDropdownMenu();
 }
 
-// -----------------display all episodes-----------------------------
+// -----------------Display all shows-----------------------------
 
-function makePageForEpisodes(episodeList) {
+const imageNotAvailable =
+  "https://sainfoinc.com/wp-content/uploads/2018/02/image-not-available.jpg";
+
+function makePageForShows(showList) {
   rootElem.innerHTML = "";
-  episodeList.forEach((episode) => createCard(episode));
-  episodesCount.innerText = `Displaying ${episodeList.length}/${allEpisodes.length}`;
+
+  showCount.innerText = `Displaying ${showList.length} shows`;
+  showCount.style.color = "white";
```

Comment:
You could create a class and add that instead meaning you have more control over the class later and don't need to add 101 things to the JS :)